sdk/PowerBI.Api/PowerBI.Api.csproj

```diff
@@ -76,9 +145,7 @@
       <Private>True</Private>
     </Reference>
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Source\" />
```

Comment:
someone did it in one commit, now csproj has flat list of files.
I will revert it later.